indexer/scripts/elastic-conf.py

```diff
@@ -54,6 +66,12 @@ def define_options(self, ap: argparse.ArgumentParser) -> None:
             default=os.environ.get("ELASTICSEARCH_SNAPSHOT_REPO") or "",
             help="ES snapshot repository name",
         )
+        ap.add_argument(
+            "--es-snapshot-type",
+            dest="es_snapshot_type",
+            default=os.environ.get("ELASTICSEARCH_SNAPSHOT_REPO_TYPE") or "fs",
+            help="ES snapshots type, default fs",
+        )
```

Comment:
```suggestion
        ap.add_argument(
            "--es-snapshot-repo-type",
            dest="es_snapshot_repo_type",
            default=os.environ.get("ELASTICSEARCH_SNAPSHOT_REPO_TYPE") or "",
            help="ES snapshots type, default fs",
        )
```

If we have to set a default value, it makes sense to set it in one place only... either in `deploy.sh` or here but not both.